Sasquatch/Sasquatch/ViewControllers/MSMainViewController.swift

```diff
@@ -109,6 +110,7 @@ class MSMainViewController: UITableViewController, AppCenterProtocol, CLLocation
 
   override func viewWillAppear(_ animated: Bool) {
     super.viewWillAppear(animated)
+    self.setAppSecretButton.isEnabled = startUpModeForCurrentSession == 1 ? false : true
```

Comment:
`1` is a magic number. Can we refactor this to `startUpModeForCurrentSession == startUpMode$WHATEVEROURDESIREDVALUEUS ? false : true`?

Reply:
Yes we should have a constant or enum if multiple values to explicitly define the startUpModes because just 1 is not really explicit.